tsdownsample/downsampling_interface.py

```diff
@@ -143,24 +143,28 @@ def downsample(self, *args, n_out: int, **kwargs):  # x and y are optional
 class AbstractRustDownsampler(AbstractDownsampler, ABC):
     """RustDownsampler interface-class, subclassed by concrete downsamplers."""
 
-    def __init__(self, resampling_mod: ModuleType):
-        super().__init__(_rust_dtypes, _y_rust_dtypes)  # same for x and y
-        self.rust_mod = resampling_mod
+    @property
```

Comment:
do we put the properties above or below the `__init__`

Reply:
I would do below the __init__